src/js/view/preview.js

```diff
@@ -1,13 +1,53 @@
 define([
-    'utils/underscore'
-], function(_) {
+    'utils/underscore',
+    'utils/helpers'
+], function(_, utils) {
 
     var Preview = function(_model) {
         this.model = _model;
 
-        this.model.on('change:playlistItem', this.loadImage, this);
+        this.model.on('change:playlistItem', onPlaylistItem, this);
+        this.model.on('change:mediaModel', onMediaModel, this);
     };
 
+    function onChangeState(model, state) {
+        if (state === 'complete') {
+            this.loadImage(model, model.get('playlistItem'));
+        }
+    }
+
+    function onMediaModel() {
+        this.model.mediaModel.off('change:mediaType', onMediaType);
+        this.model.mediaModel.on('change:mediaType', onMediaType, this);
+    }
+
+    function onMediaType(mediaModel) {
+        var audio = (mediaModel.get('mediaType') === 'audio');
+        if (audio) {
+            this.loadImage(this.model, this.model.get('playlistItem'));
+        }
```

Comment:
We want to alway change the backgroundImage property. When it's audio set it to the poster, otherwise clear it.